### ISSUE TITLE
【依頼】フリマアプリ挙動確認

### ISSUE THREAD

**Issue by HilotoTaketake** [HilotoTaketake/furima_27497#10]
自身のフリマアプリのURL
https://furima-27497.herokuapp.com/

Basic認証のIDとパスワード
BASIC_AUTH_PASSWORD:      Thurima711
BASIC_AUTH_USER:          takenishi711

出品者用のメールアドレスとパスワード
メールアドレス: blue@blue
パスワード: blue11

購入者用のメールアドレスとパスワード
メールアドレス: white@white
パスワード: white1

購入用カードの番号・期限・セキュリティコード
番号: 4242424242424242
期限: 12/24
セキュリティコード: 123

**Comment by naru6117** [comment]
竹西さん、ご提出ありがとうございます！
挙動確認させて頂いたところ、以下の不備が見受けられましたので修正をお願いします！
### [修正箇所]
**①Basic認証のアイパスを入力しても、アクセスできないので、ご確認お願いします(^^)**
![スクリーンショット 2020-08-05 15 39 40](https://user-images.githubusercontent.com/49669977/89380079-e6933d80-d731-11ea-9c40-e9bcea43aa88.png)


修正し次第、以下の最終課題完了報告フォームを再度提出お願いします！
https://forms.gle/dYuR1wjeHepbEhU7A

**Comment by HilotoTaketake** [comment]
確認ありがとうございます。
3度ほど送信したパスワードユーザネームを使用してログインしてみたのですが無事入れましたのでもう一度ご確認いただきたいです。
タイプをして入れなければコピー&ペーストで試してもらいたいです。
お手数ですがよろしくお願いします。

**Comment by mutorisa** [comment]
ご提出ありがとうございます！
再度確認させて頂きましたが、やはりログインできないようです（タイプ及びコピペ実施済み）
念の為ご確認ですが、記載あるとおり以下で間違いはありませんでしょうか？
**Basic認証のIDとパスワード**
**BASIC_AUTH_PASSWORD: Thurima711**
B**ASIC_AUTH_USER: takenishi711**

再度ご確認をお願いします。
もし、悩むようであれば一度コネクトにてメンターに相談してみましょう！

修正し次第、以下の最終課題完了報告フォームを再度提出お願いします！
https://forms.gle/dYuR1wjeHepbEhU7A

**Comment by HilotoTaketake** [comment]
はい、パスワードがThurima711、ユーザネームがtakenishi711で間違いありません。わかりました、一度メンターさんに確認してもらいます。17:00-18:00まで面談なので一時間後に再度提出させてもらいますのでご確認よろしくお願いします。

**Comment by HilotoTaketake** [comment]
お疲れ様です！
上記の情報修正いたしました！
パスワードとユーザネームの位置が逆だったようです。
お手数ですが再度ご確認よろしくお願いします。

**Comment by mutorisa** [comment]
再度ご提出ありがとうございます！
挙動確認させて頂いたところ、ログインは問題なくできました✨
ただし、修正していただきたい箇所が見つかりましたので、ご確認をお願いします！

**[修正依頼箇所]**
**○記載していただいた、出品者用のアドレス及びパスワードと購入者用のアドレス及びパスワードにてログインすることができませんでした。**
**ログインができるアドレス及びパスワードの記載をお願いします！**
（ログインするとこのようにエラーが出てしまいました）
<img width="443" alt="スクリーンショット 2020-08-05 18 48 40" src="https://user-images.githubusercontent.com/65144927/89400388-cb82f680-d74e-11ea-9400-3cf4ce7f26ba.png">


**○出品済みの商品を見ることができない為、ログインしていないユーザーもログインしているユーザーも商品を閲覧できるようにしましょう。**

**○商品出品後、商品の説明が表示されていないので、表示されるようにしましょう。**

**○商品を出品する際、画像が０枚でも出品できてしまう為、画像が０枚の時は出品できないよう実装しましょう。**

**○クレジット情報及び送付先情報を入力しても、商品の購入ができませんでした。正常に商品を購入できるようにしましょう。**
（商品購入後の必須項目の確認はできておりません。次回提出時に併せてご確認させていただきます。）

**○必須項目ではないですが、商品を出品する際のカテゴリーが「メンズ」ではなく「メンス」になってしまっていた為、変更されると良いかもしれません。また、商品発送日の目安は「日数（１日〜２日等）」を入れていただいたほうがわかりやすいように感じました。**

修正し次第、以下の最終課題完了報告フォームを再度提出お願いします！
https://forms.gle/dYuR1wjeHepbEhU7A

**Comment by HilotoTaketake** [comment]
自身のフリマアプリのURL
https://furima-27497.herokuapp.com/

Basic認証のIDとパスワード
BASIC_AUTH_USER: takenishi711
BASIC_AUTH_PASSWORD: Thurima711

出品者用のメールアドレスとパスワード
メールアドレス: kiwi@kiwi
パスワード: kiwi11

購入者用のメールアドレスとパスワード
メールアドレス: kani@kani
パスワード: kani11

購入用カードの番号・期限・セキュリティコード
番号: 4242424242424242
期限: 12/31
セキュリティコード: 123

**Comment by mutorisa** [comment]
再度ご提出ありがとうございます！
ご確認させていただいたところ、
>○記載していただいた、出品者用のアドレス及びパスワードと購入者用のアドレス及びパスワードにてログインすることができませんでした。


>○必須項目ではないですが、商品を出品する際のカテゴリーが「メンズ」ではなく「メンス」になってしまっていた為、変更されると良いかもしれません。また、商品発送日の目安は「日数（１日〜２日等）」を入れていただいたほうがわかりやすいように感じました。


上記２点については修正の確認ができました。

ただし、**他の項目については修正の確認ができておりません。**
**商品出品後、商品を見ようとしたところ、以下添付画像のようなエラーが出てしまいました。今一度ご確認をお願いします。また、出品商品についてですが、企業への提出物となりますので、動物ではなく実際のフリマアプリに出品されているような商品（洋服等）を出品された方が無難かと思います。よろしくお願いします**
<img width="553" alt="スクリーンショット 2020-08-06 16 53 16" src="https://user-images.githubusercontent.com/65144927/89506349-efedda00-d805-11ea-8ea9-2464b1b5d018.png">


修正し次第、以下の最終課題完了報告フォームを再度提出お願いします！
https://forms.gle/dYuR1wjeHepbEhU7A

**Comment by HilotoTaketake** [comment]
ご確認ありがとうございます。
出品した商品が確認できないとのことですね。
先ほど提出前にメンターさんに繋いでもらって挙動を確認してもらってその点は何も異常がなかったので再度メンターさんに繋いで確認をとって提出します。

**Comment by HilotoTaketake** [comment]
自身のフリマアプリのURL
https://furima-27497.herokuapp.com/

Basic認証のIDとパスワード
BASIC_AUTH_USER: takenishi711
BASIC_AUTH_PASSWORD: Thurima711

出品者用のメールアドレスとパスワード
メールアドレス: kiwi@kiwi
パスワード: kiwi11

購入者用のメールアドレスとパスワード
メールアドレス: kani@kani
パスワード: kani11

購入用カードの番号・期限・セキュリティコード
番号: 4242424242424242
期限: 12/31
セキュリティコード: 123

**Comment by mo1015** [comment]
再提出ありがとうございます！
挙動確認させて頂いたところ、以下の不備が見受けられましたので修正をお願いします！

[修正箇所]
**○Basic認証のアイパスを入力しても、アクセスできなくなっております**
再度、本番環境で実装できているか確認をお願いします！

修正でき次第、以下の最終課題完了報告フォームを再度提出お願いします！
https://forms.gle/dYuR1wjeHepbEhU7A

**Comment by HilotoTaketake** [comment]
ありがとうございます。こちらで本番環境を２つ用意して確認して認証は大丈夫なのですがメンターさん側ではエラーになっているのですね。
再度メンターさんにビデオ通話で確認をとって無事ログインできているので提出いたします。

パスワードのスペル(Thurima711)をお確かめの挙動を確認お願いいたします。
自身のフリマアプリのURL
https://furima-27497.herokuapp.com/

Basic認証のIDとパスワード
BASIC_AUTH_USER: takenishi711
BASIC_AUTH_PASSWORD: Thurima711

出品者用のメールアドレスとパスワード
メールアドレス: kiwi@kiwi
パスワード: kiwi11

購入者用のメールアドレスとパスワード
メールアドレス: kani@kani
パスワード: kani11

購入用カードの番号・期限・セキュリティコード
番号: 4242424242424242
期限: 12/31
セキュリティコード: 123
確認なのですがコピペをしても認証が効かない状態でしょうか。お確かめ願います。

**Comment by mo1015** [comment]
竹西さん、再提出ありがとうございます！
挙動確認させて頂きましたが、今回は問題なく**ログインができました**！
また、**出品した商品の詳細も確認できております**のでLGTMとさせていただきます:tada:
おめでとうございます🎉お疲れさまでした！

**※確認したのは必須機能のみで追加実装した機能については確認していません。
そのため追加機能を実装している場合は、正しく想定通りの挙動であるかご自身で再度確認をお願いいたします。**

**Comment by HilotoTaketake** [comment]
嬉しいです！
ありがとうございました。
